### DISCUSSION TITLE
Push new versions to addon stores

### DISCUSSION THREAD

**Discussion by r3dicalnusz** [trakt-tools/universal-trakt-scrobbler#403] [Q&A]
How could we get to a point where the releases in the browser stores are updated frequently?
Or how could we get a update check function for the manual installs?

I want the auto update from the store versions but I don't want to wait months to get the fixes.

**Comment by r3dicalnusz** [comment]
@MrMamen updated the version to 0.4.1 2 weeks ago. There needs to be some kind of automation to push things like that to the addon stores.

**Comment by MrMamen** [comment]
It's a bug. Could you look at it @rafaelgomesxyz ?

**Reply by r3dicalnusz** [reply]
Looks like he can't. What now?

**Reply by rafaelgomesxyz** [reply]
Sorry! I've been busy with my job again. I've pushed the new version manually for now, it's in the review process by Google.

**Reply by r3dicalnusz** [reply]
Were you able to find out what stopped the automation?

**Reply by rafaelgomesxyz** [reply]
It's some authentication issue, maybe a token expired or Google changed something in their API. I haven't fixed it yet, so I'll continue to push manually until I have some time to fix the automation.
